queries.sql

```diff
@@ -169,4 +169,122 @@ vet_clinic-# animals.owner_id = owners.id group by owners.full_name order by cou
      2 | Bob
      2 | Jennifer Orwell
      1 | Sam Smith
-(5 rows)
\ No newline at end of file
+(5 rows)
+
+
+vet_clinic=# select visits.date_of_visit, animals.name, vets.name from animals
+vet_clinic-# join visits on animals.id = visits.animals_id
+vet_clinic-# join vets on visits.vet_id = vets.id
+vet_clinic-# where vets.name = 'William Tatcher'
+vet_clinic-# order by visits.date_of_visit desc limit 1;
+ date_of_visit |  name   |      name
+---------------+---------+-----------------
+ 2021-01-11    | Blossom | William Tatcher
+(1 row)
+
+
+vet_clinic=# select visits.date_of_visit, animals.name, vets.name from animals
+vet_clinic-# join visits on animals.id = visits.animals_id
+vet_clinic-# join vets on visits.vet_id = vets.id
+vet_clinic-# where vets.name = 'William Tatcher'
+vet_clinic-# order by visits.date_of_visit desc limit 1;
+ date_of_visit |  name   |      name
+---------------+---------+-----------------
+ 2021-01-11    | Blossom | William Tatcher
+(1 row)
+
+
+vet_clinic=# select vets.name, species.name from vets
+vet_clinic-# left join specializations on specializations.vet_id = vets.id
+vet_clinic-# left join species on specializations.species_id = species.id;
+       name       |                         name
+------------------+------------------------------------------------------
+ William Tatcher  | Pokemon
```

Comment:
- [ ] Good job working on this, but please as stated above kindly include only the queries and not the terminal prompt and output

Reply:
@AnselemOdims I have removed all of the command prompt outputs.